### PR TITLE
Added an Orbitting Circles Animated Page

### PR DESCRIPTION
**Pull request by SOMNATH0904**
Thank you for making a pull request to ZTM's Animation Nation project. We will review your pull request as soon as we can, but keep in mind we can get very busy during Hacktoberfest.

While you're waiting, please look over this checklist. Feel free to write a description of what you did as well!

## What I did
I have created an Animated page where three circles are moving around with each other in circle way. It is created using only HTML and CSS files.

*Optional: Edit this to tell us what you did!*

## Pull request checklist

- [x] This pull request uses **HTML and CSS** only.
- [x] All files in this pull request are in a directory named: `<github_username>-<art_name>` without the <>.
- [x] Your pull request contains only `index.html` and `styles.css`.
- [x] Your html file is named exactly: `index.html`
- [x] Your css file is named exactly: `styles.css`
- [x] Your submission includes **at least one animation**.
- [x] Your pull request **does not modify any other files** in the repository.

#### Not normally needed
- [ ] Check this if you would like to discuss an improvement besides adding animations

**Note: if your pull request makes an improvement besides adding a CSS animation you will need to discuss this with the maintainers. You can help kickstart this process by explaining what your change does and why you are making it in the "What I did" section**

## What to expect

- If your pull request meets the checklist requirements it will be approved and merged by a maintainer
- If your pull request needs some changes. A maintainer will leave a comment requesting changes
- If your pull request significantly violates the criteria in the checklist it will be closed with feedback.

### CODE REVIEW

File: Art/somnath0904-OrbittingCircles/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Orbiting Circles</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css">
+  <link rel="stylesheet" href="styles.css">
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js"></script>
```
Comment (stilt0n) [inline Art/somnath0904-OrbittingCircles/index.html:11]:
```suggestion
  <link rel="stylesheet" href="styles.css">
```
You do not seem to be using bootstrap, jquery, or popperjs. Can you remove the links and script tags to those libraries?

**Review by stilt0n — CHANGES_REQUESTED** [review]
Looks pretty good but can you remove the links to the unused libraries? After you do that I think this will be good to merge

**Review by stilt0n — APPROVED** [review]
Thanks. This looks ready to merge now.

**Comment by SOMNATH0904** [comment]
@stilt0n Can you please give the label "hacktoberfest-accepted", so that it will be reflected in the hactoberfest site.
